### PR TITLE
Serializers for kved and pep.is_pep

### PR DESCRIPTION
**Pull request by Andrii17s**
Connected with https://github.com/3v-workspace/DataOcean/pull/195 and https://github.com/3v-workspace/DataOcean/pull/191

### CODE REVIEW

File: business_register/serializers/kved_serializers.py [hunk 1]
```diff
@@ -10,6 +10,11 @@ class KvedDetailSerializer(DynamicFieldsMixin, serializers.ModelSerializer):
...
 
+    section_en = serializers.CharField(source='section.name_en')
+    division_en = serializers.CharField(source='division.name_en')
+    group_en = serializers.CharField(source='group.name_en')
+
     class Meta:
         model = Kved
-        fields = ['id', 'code', 'name', 'name_en', 'group', 'division', 'section']
+        fields = ['id', 'code', 'name', 'name_en', 'group', 'division', 'section',
+                  'section_en', 'division_en', 'group_en']
```
Comment (roman-tiukh) [inline business_register/serializers/kved_serializers.py:20]: incorrect sequence of field

File: business_register/serializers/sanction_serializers.py [hunk 2]
```diff
@@ -29,6 +29,11 @@ class PersonSanctionSerializer(DynamicFieldsMixin, serializers.ModelSerializer):
     types_of_sanctions = serializers.StringRelatedField(
         many=True, help_text=SanctionType._meta.get_field('name').help_text
     )
+    is_pep = serializers.BooleanField(
+        source='pep.is_pep', default=False,
```
Comment (roman-tiukh) [inline business_register/serializers/sanction_serializers.py:33]: maybe better `default=None`

**Review by roman-tiukh — CHANGES_REQUESTED** [review]

File: business_register/serializers/sanction_serializers.py [hunk 3]
```diff
@@ -28,6 +28,10 @@ class PersonSanctionSerializer(DynamicFieldsMixin, serializers.ModelSerializer):
     types_of_sanctions = serializers.StringRelatedField(
         many=True, help_text=SanctionType._meta.get_field('name').help_text
     )
+    is_pep = serializers.BooleanField(
+        source='pep.is_pep', default=None,
+        help_text='Чи є особа під санкціями публичним діячем України'
```
Comment (roman-tiukh) [inline business_register/serializers/sanction_serializers.py:33]: 1. translate to English, need for API documentation
2. remove "України" from help_text

**Review by roman-tiukh — CHANGES_REQUESTED** [review]

File: business_register/serializers/sanction_serializers.py [hunk 4]
```diff
@@ -1,3 +1,4 @@
+from django.utils.translation import ugettext_lazy as _
```
Comment (roman-tiukh) [inline business_register/serializers/sanction_serializers.py:1]:
```suggestion
from django.utils.translation import gettext_lazy as _
```

**Review by roman-tiukh — APPROVED** [review]

**Review by roman-tiukh — APPROVED** [review]

**Review by AlenaYanish — APPROVED** [review]

**Pull request merged**